Packs/VMRay/TestPlaybooks/playbook-VMRay-test-File-Enrichment.yml

```diff
@@ -0,0 +1,530 @@
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: b3e07aeb-6572-4e4f-8a05-c2df5913cae5
+    type: start
+    task:
+      id: b3e07aeb-6572-4e4f-8a05-c2df5913cae5
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "9"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "7":
+    id: "7"
+    taskid: 566fdba9-a9b2-43fd-8a63-d3760139027a
+    type: title
+    task:
+      id: 566fdba9-a9b2-43fd-8a63-d3760139027a
+      version: -1
+      name: End of test
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 930
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "9":
+    id: "9"
+    taskid: 8a2b65af-635b-40a0-82fb-fd6e8699bb27
+    type: regular
+    task:
+      id: 8a2b65af-635b-40a0-82fb-fd6e8699bb27
+      version: -1
+      name: Delete context
+      description: Delete field from context
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "12"
+      - "19"
+      - "22"
+    scriptarguments:
+      all:
+        simple: "yes"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "12":
+    id: "12"
+    taskid: f460de52-2b6f-4e1c-82b2-7a5df117436d
+    type: playbook
+    task:
+      id: f460de52-2b6f-4e1c-82b2-7a5df117436d
+      version: -1
+      name: Valid MD5
+      description: Test file enrichment with a valid MD5 hash
+      playbookName: File Enrichment - VMRay
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "18"
+    scriptarguments:
+      MD5:
+        complex:
+          root: File
+          accessor: MD5
+      SHA1:
+        complex:
+          root: File
+          accessor: SHA1
+      SHA256:
+        complex:
+          root: File
+          accessor: SHA256
+    separatecontext: true
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 0,
+          "y": 370
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "15":
+    id: "15"
+    taskid: 5a0f5737-e7ef-41de-808c-5abc69c5ca82
+    type: playbook
+    task:
+      id: 5a0f5737-e7ef-41de-808c-5abc69c5ca82
+      version: -1
+      name: Invalid MD5
```

Comment:
Please add the condition that the playbook won't stop on errors, as we discussed